indicators/static_analysis_common_vulnerabilities.json

```
@@ -0,0 +1,22 @@
+{
+    "@context": "https://w3id.org/everse/rsqi#",
+    "@id": "https://w3id.org/everse/i/indicators/static_analysis_common_vulnerabilities",
+    "@type": "rs:SoftwareQualityIndicator",
+    "name": "has static analysis for common vulnerabilities",
+    "description": "At least one static code analysis tool used includes rules or techniques to detect common vulnerabilities specific to the language or environment.",
+    "keywords": ["security", "vulnerabilities", "static analysis"],
+    "identifier": "https://w3id.org/everse/i/indicators/static_analysis_common_vulnerabilities",
+    "status": "Active",
+    "version": "1.0.0",
+    "author": {
+        "@type": "Organization",
+        "name": "OpenSSF"
+    },
+    "contactPoint": {
+        "@type": "schema:Person",
+        "name": "Thomas Vuillaume",
+        "email": "thomas.vuillaume@lapp.in2p3.fr"
+    },
+    "qualityDimension": "https://w3id.org/everse/i/dimensions/security",
+    "dateCreated": "03-04-2025"
```

key has been updated to "created"
```suggestion
    "created": "03-04-2025"
```